docs/rackhd/event_notification.rst

```diff
@@ -1,105 +1,264 @@
 Event Notification
 ------------------
 
-The change of resources managed by RackHD could be retrieved from AMQP messages.
-
-- Exchange: **on.events**
-- Routing Key **event.<type>**
+RackHD support event notification via both AMQP and web hook.
 
-`<type>` is event's type, it's the same with event's payload `type`, see payloads_ for more details.
+Web hook allow user's application to subscribe certain RackHD published events published by configured URL, when one of the subscribed events is triggered, RackHD will send a HTTP POST request with payload to configured URL.
 
-An example of listening node events using tool 'sniff.js' located at https://github.com/RackHD/on-tools/blob/master/dev_tools/README.md.
+AMQP allow users don't need to register webhook URL, all RackHD defined events will be published, but users need to filter events in user side.
```

Comment:
recommended language update:
"AMQP allow users don't need to register webhook URL, all RackHD defined events will be published, but users need to filter events in user side." --> 

"RackHD also publishes defined events over AMQP, so subscribers to RackHD's instance of AMQP don't need to register a webhook URL to get events. The AMQP events can be prolific, so we recommend that consumers filter events as they are received to what is desired."